reactular.tsx

```diff
@@ -9,7 +9,8 @@ type OnChanges<T> = {
 export const reactular = <Props extends object>(
   Component: React.ComponentType<Props>,
   bindingNames: Array<keyof Props> = [],
-  wrapper?: string | React.ComponentType,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  wrapper?: string | React.FunctionComponent<React.PropsWithChildren<any>>,
```

Comment:
We don't pass props to the wrapper component, so this could probably be `{}` or maybe even `unknown`? (I think they end up being equivalent in this context.)

```suggestion
  wrapper?: string | React.FunctionComponent<React.PropsWithChildren<{}>>,
```

Reply:
Will update 👍🏻